src/mjolnir/pbfgraphparser.cc

```diff
@@ -1662,6 +1662,10 @@ struct graph_callback : public OSMPBF::Callback {
     osmdata_.max_changeset_id_ = std::max(osmdata_.max_changeset_id_, changeset_id);
   }
 
+  virtual void header_callback(const uint64_t timestamp, const uint64_t sequence_number, const std::string& base_url) override {
+    std::cout << "header " << timestamp << " " << sequence_number << " " << base_url << std::endl;
```

Comment:
i feel like we could merge this pr if we changed this to a LOG or  TODO and noted that we are setting ourselves up to use this in the future if we need to. what do you think @gknisely ?